core/src/main/java/org/opensearch/sql/storage/Table.java

```diff
@@ -29,6 +30,8 @@ public interface Table {
    */
   PhysicalPlan implement(LogicalPlan plan);
```

Comment:
May want to get rid of this eventually

Reply:
Reason..is there a better way of doing this? Has anything been proposed on this?

Reply:
Oh I was referring to the line above that doesn't use the PlanContext. All implement calls should use PlanContext.